docs/src/documentation/06-kiwi-use/02-content/04-specific-areas/social-media/02-visual-treatment.mdx

```diff
@@ -14,12 +15,15 @@ Our photography should always have a sense of optimism.
 The subjects and environments should feel authentic.
 We shouldn't be afraid of showing **real faces and emotions.**
 People should be **unposed** and their **composition** within the environment **natural**.
-Lighting, color and tone should be **unfiltered and realistic \#NoFilter!**
+Lighting, color, and tone should be **unfiltered and realistic \#NoFilter!**
 
 Each image should establish an emotional connection with our audience,
 showing them the world Kiwi.com can offer.
 
+<!-- prettier-ignore-start -->
+<!-- eslint-disable-next-line -->
 ## Our destinations...
+<!-- prettier-ignore-end -->
 
 **ARE** real, authentic and full of wanderlust.
```

Comment:
Vale warnings complain about words like "ARE" and "NOT", and considering that they are already in bold, no need for all caps, but your choice 😉

Reply:
I also disliked the warnings (and the style before even seeing the warnings). But I didn't create the text, so I felt bad adjusting it. But if Vale says it, it must be true, so I I guess I should change it.